### ISSUE TITLE
Error in mlb_vqa if no valid examples in valid set

### ISSUE THREAD

**Issue by backpropper** [facebookresearch/ParlAI#756]
It seems like there is a bug in checking the valid accuracy of the mlb_vqa model. The model goes through the first pass smoothly checking all the valid examples in the valid set but in the second pass, it outputs a total of "0" examples.

Command to reproduce behavior:
`
python examples/train_model.py -m mlb_vqa -t pytorch_teacher --pytorch-buildteacher vqa_v1 -mf mlb --dataset parlai.tasks.vqa_v1.agents -bs 128 -im resnet152_spatial --image-size 448 --image-cropsize 448 -vtim 120 -eps 50
`

**Comment by klshuster** [comment]
Closed with #761 - let me know if you have any more questions!
